api/user/schema.py

```diff
@@ -175,24 +176,27 @@ def mutate(self, info, email, **kwargs):
         query_user = User.get_query(info)
         active_user = query_user.filter(UserModel.state == "active")
         exact_user = active_user.filter(UserModel.email == email).first()
+
         if not exact_user:
             raise GraphQLError("User not found")
 
         new_role = RoleModel.query.filter_by(id=kwargs['role_id']).first()
+
         if not new_role:
             raise GraphQLError('invalid role id')
 
         current_user_role = exact_user.roles[0].role
         if new_role.role == current_user_role:
             raise GraphQLError('This role is already assigned to this user')
 
+        restricting_admin(new_role.role)
```

Comment:
@cop1fab Please change the name of this function to `check_andmin_restiction` or something in form of action.

Reply:
done